api/v1alpha1/ratelimit_types.go

```diff
@@ -42,17 +54,45 @@ type GlobalRateLimit struct {
 	Rules []RateLimitRule `json:"rules"`
 }
 
+// LocalRateLimit defines local rate limit configuration.
+type LocalRateLimit struct {
+
+	// Envoy requires a default rate limit to be set for each route.
```

Comment:
this is not relevant to the end user, can this be rewritten with the app dev in mind ?

Reply:
This comment is just for EG developers. It won't show up in the generated CRD yaml and md file.

Reply:
it should then probably live in the design doc or translation lib, not in the API file